tests/staggered_invert_test.cpp

```diff
@@ -170,6 +163,11 @@ int main(int argc, char **argv)
   QudaEigParam mg_eig_param[mg_levels];
 
   if (inv_multigrid) {
+
+    // Set some default values for MG solve types
+    setQudaMgSolveTypes();
+    reliable_delta = 1e-4;
```

Comment:
Do we want that to be hard-coded? Looks like it should be an option @weinbe2 ?

Reply:
Fixed